tools/sync/main.go

```diff
@@ -20,38 +20,51 @@ func main() {
 
 	ctx := context.Background()
 
+	sync := initLogging()
+	defer sync()
+	logger := log.Scoped("sync", "tool to sync contributors")
+
+	// Read config and parse our contributors
 	conf, err := clabot.ParseConfig()
 	if err != nil {
-		println(err.Error())
-		os.Exit(1)
+		logger.Fatal(err.Error())
 	}
 	existingHandles := make(map[string]struct{}, len(conf.Contributors))
 	for _, handle := range conf.Contributors {
 		existingHandles[handle] = struct{}{}
 	}
 
+	// List CLA form responses
 	resps, err := responses.ListResponses(ctx, *pages)
 	if err != nil {
 		// EOF indicates we hit maximum pages
 		if err != io.EOF {
-			println(err.Error())
-			os.Exit(1)
+			logger.Fatal(err.Error())
 		}
-		fmt.Printf("Reached maximum pages %d\n", *pages)
+		logger.Info("reached maximum pages", log.Int("pages", *pages))
 	}
-	fmt.Printf("Found %d responses\n", len(resps))
+	logger.Info("listed responses", log.Int("responses", len(resps)))
+
+	// For each response, if not yet in config, add it
+	var added int
 	for _, resp := range resps {
-		// If not yet in config, add it
 		if _, exists := existingHandles[resp.GitHubHandle]; !exists {
-			fmt.Printf("Adding contributor %q (%s)\n", resp.GitHubHandle, resp.Name)
+			logger.Info("adding contributor",
+				log.String("gitHubHandle", resp.GitHubHandle),
```

Comment:
```suggestion
				log.String("githubHandle", resp.GithubHandle),
```

Reply:
It's spelled GitHub I think 😛 

<img width="539" alt="image" src="https://user-images.githubusercontent.com/23356519/185693675-bb6556cb-3f4d-4655-b67d-e226320fb33a.png">

Reply:
ROFL 🤣 Don't mind me